microprofile-openapi/src/test/java/org/wildfly/quickstart/microprofile/openapi/OpenAPIContextIT.java

```diff
@@ -47,9 +48,8 @@ public void testOpenAPIContextIsAvailable() throws IOException, InterruptedExcep
         final HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
```

Comment:
This is not portable.  If the OpenAPI request does not define a specific encoding, via the Accept-Charset (or Accept) header, then it will use UTF-8 was specified in the request.  Since the above request does include this header, the response may differ from the default charset of the test client.

Either:

- Specify an Accept-Charset or Accept header in the request (using the default charset of the test client)
- Read the body using default encoding of the OpenAPI endpoint, i.e. HttpResponse.BodyHandlers.ofString(StandardCharsets.UTF_8)